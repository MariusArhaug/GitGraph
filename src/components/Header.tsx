import React from 'react'

export function Header() {
  return (
    <header className="text-center py-16 mt-40">
      <h1 className="text-5xl">GitGraph Data</h1>
      <h3 className="text-2xl mt-10">Hello, here is data:</h3>
    </header>
  )
}
