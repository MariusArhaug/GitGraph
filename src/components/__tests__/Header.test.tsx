import React from 'react';
import renderer from 'react-test-renderer';
import { Header } from '../Header';

it('reders correctly', () => {
    const tree = renderer
        .create(<Header/>)
        .toJSON();
    expect(tree).toMatchSnapshot()
})