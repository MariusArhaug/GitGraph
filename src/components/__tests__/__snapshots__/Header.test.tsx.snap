// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reders correctly 1`] = `
<header>
  <h1
    className="text-center text-5xl py-16"
  >
    "Placeholder title"
  </h1>
</header>
`;
